tools/integration_tests/util/setup/setup.go

```diff
@@ -116,7 +116,12 @@ func CreateTempFile() string {
 	if err != nil {
 		LogAndExit(fmt.Sprintf("Error in the opening the file %v", err))
 	}
-	defer file.Close()
+
+	defer func() {
```

Comment:
We should use the CloseFile defined in operations. If you are not using it because of cyclic dependency between setup.go and *_operations.go, then we shouldn't have operations package dependent on setup.